nautobot_golden_config/nornir_plays/config_backup.py

```diff
@@ -45,6 +45,10 @@ def run_backup(  # pylint: disable=too-many-arguments
     obj = task.host.data["obj"]
     settings = device_to_settings_map[obj.id]
 
+    if not settings.backup_enabled:
```

Comment:
I dont see this code block ever being hit.  I think you want to look at the `verify_settings` call that happens in the entrypoint method in this case `run_backup()` 

<img width="1160" alt="Screenshot 2025-02-26 at 7 51 50 PM" src="https://github.com/user-attachments/assets/2bc49f25-ae8f-47cf-8838-67461acba016" />

If I set the backup repository + backup path but don't set the boolean to enable backups, then I get this error message, but I don't think that is the intention.